android/battleBuddyAndroid/components/common/Card.js

```diff
@@ -3,6 +3,7 @@ import styled, { css } from 'styled-components/native';
 import PropTypes from 'prop-types';
 
 const StyledCard = styled.ImageBackground`
+  /* prettier-ignore */
```

Comment:
Ignore the ignore comments for now :P 
It's an issue with one of my linters/formatters that keep converting the camelCase names to lowercase. Will look into a fix once I integrate prettier and eslint